pkg/migrate/cstor/volume.go

```diff
@@ -964,6 +960,38 @@ retry:
 	return nil
 }
 
+// removePodAffinity removes the affinity from target deployment
+// to verify volume health as the application is scaled down and pod
+// can be schedules according to the rules
+func (v *VolumeMigrator) removePodAffinity() error {
+	cvp, err := v.OpenebsClientset.CstorV1().
```

Comment:
Even in restart cases also policy will be there till  `openebs.io` group CV exists.

Reply:
yes